src/commands/packageBuilder.ts

```diff
@@ -50,6 +50,23 @@ export function getMembers(metadataTypes: string[], retrieveManaged?: boolean):
                         
                     })
                     .catch(reject);
+            } else if(r.xmlName === 'StandardValueSet') {
+                // metadata list for StandardValueSet doesn't work. This is the workaround
```

Comment:
Interesting.  Is there a link you could include here, so we could track this issue going forward?  Hardcoding stuff is always bound to get out of date, but if it's required now... it's required.   Nice work.  👏